src/components/NavigationWidget.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { connect } from '../store/Connect';
+import CrosshairToggle from './CrosshairToggle.jsx';
 import '../../sass/components/_ciq-navigation-widget.scss';
 
-import { ZoominIcon, ZoomoutIcon, HomeIcon, ScaleIcon } from './Icons.jsx';
+import { ZoominIcon, ZoomoutIcon, ScaleIcon } from './Icons.jsx';
```

Comment:
@shayan-binary-2 why is `HomeIcon` removed?

Reply:
@morteza-binary HomeIcon used to be on the right hand, after that Ui/UX decide to move it to left on the top of navigation, and now we move back to the left side